_maps/templates/battlecruiser_starfury.dmm

```diff
@@ -2648,7 +2648,7 @@
 /obj/machinery/light/directional/east,
 /obj/item/storage/belt/military,
 /obj/item/ammo_box/magazine/sniper_rounds,
-/obj/item/gun/ballistic/automatic/sniper_rifle/syndicate,
+/obj/item/gun/ballistic/rifle/sniper_rifle/syndicate,
```

Comment:
is this something that'll need an updatepaths

Reply:
oh I think I know what you're talking about. I'm not super familiar with how to do it, but it's not too hard right?

Reply:
All done 👍